dvc/command/pipeline.py

```diff
@@ -73,13 +73,53 @@ def _show_ascii(self, target, commands, outs):
         d = Dagascii(nodes, edges)
         d.draw()
 
+    def __write_dot(self, target, commands, outs, filename):
```

Comment:
There is a lot of common code with `_show_ascii`. Could you please extract it into a `_build_graph` method that is going to be called in both `_show_ascii` and `_write_dot` and is going to return `edges, nodes` pair or something like that?